plugins/vendor/zoom/ZoomDropFolderPlugin/batch/KZoomDropFolderEngine.php

```diff
@@ -102,7 +102,7 @@ protected function getMeetingsInStartTimeOrder()
 			$fromInSec = time() - self::MAX_DATE_RANGE_DAYS * self::ONE_DAY;
 		}
 
-		$toInSec = min(time(), $fromInSec + self::ONE_DAY * 30);
+		$toInSec = min(time(), $fromInSec + self::ONE_DAY);

```

Comment:
This means that we will make a least on call per day, customer that has initial content of 3 years will take to long to get its value.e